src/openloco/windows/map.cpp

```diff
@@ -1,19 +1,675 @@
+#include "../companymgr.h"
+#include "../graphics/colours.h"
+#include "../graphics/gfx.h"
+#include "../graphics/image_ids.h"
+#include "../input.h"
 #include "../interop/interop.hpp"
+#include "../localisation/FormatArguments.hpp"
+#include "../objects/interface_skin_object.h"
+#include "../objects/objectmgr.h"
 #include "../ui/WindowManager.h"
+#include "../ui/scrollview.h"
+#include "../widget.h"
 
 using namespace openloco::interop;
 
 namespace openloco::ui::windows::map
 {
+    static loco_global<int32_t, 0x00523338> _cursorX2;
+    static loco_global<int32_t, 0x0052333C> _cursorY2;
+    static loco_global<uint16_t[7], 0x004FDC4C> _word_4FDC4C;
+    static loco_global<uint16_t[7], 0x004FDC4E> _word_4FDC4E;
+    static loco_global<uint32_t, 0x0526284> _dword_526284;
+    static loco_global<uint16_t, 0x00526288> _word_526288;
```

Comment:
if you define this as a `gfx::ui_size_t` then you don't need to construct a `ui_size_t` below.